src/imageTools/probe.js

```diff
@@ -76,6 +77,8 @@ function onImageRendered (e) {
       // Draw the handles
       drawHandles(context, eventData, data.handles, color);
 
+      if (config.disableTextBox) return;
+
```

Comment:
In v2, I don't remember if it is a given that `config` will be defined. You may want to updated this to `config && config.disableTextBox`